docs/modules/spark-k8s/partials/supported-versions.adoc

```diff
@@ -3,6 +3,7 @@
 // Stackable Platform documentation.
 // Please sort the versions in descending order (newest first)
 
-- 3.5.0 (Hadoop 3.3.4, Scala 2.12, Python 3.11, Java 11)
-- 3.4.1 (Hadoop 3.3.4, Scala 2.12, Python 3.11, Java 11)
-- 3.4.0 (Hadoop 3.3.4, Scala 2.12, Python 3.11, Java 11) (deprecated)
+- 3.5.0 (Hadoop 3.3.4, Scala 2.13, Python 3.11, Java 11)
+- 3.4.2 (Hadoop 3.3.4, Scala 2.13, Python 3.11, Java 11) (LTS)
+- 3.4.1 (Hadoop 3.3.4, Scala 2.13, Python 3.11, Java 11) (deprecated)
```

Comment:
Where does the Scala version come from?

Reply:
i had it confused with kafka...spark is 2.12...reverted https://github.com/stackabletech/spark-k8s-operator/pull/357/commits/7b995835d063eaf5195715b902e3ee995a82b23c